docs/docs/examples/storage/aws/s3/CONFLUENCE.md

```diff
@@ -0,0 +1,68 @@
+# AWS S3 Attachments Storage
+
+Since 8.1.0 Confluence supports storing attachments in AWS S3. To enable this feature, define bucket name and AWS region in `confluence.s3AttachmentsStorage`, for example:
+
+```yaml
+confluence:
+  s3AttachmentsStorage:
+    bucketName: confluence-attachments-bucket
+    bucketRegion: us-east-1
```

Comment:
```suggestion
tag: 8.1.0
confluence:
  s3AttachmentsStorage:
    bucketName: confluence-attachments-bucket
    bucketRegion: us-east-1
```

Reply:
@bianchi2 since the helm charts are configured to install the latest LTS by default which in Confluence's case is now 7.19.5 do you think we need to include a point here about changing the tag in values.yaml to 8.1.0 to use this feature?

Reply:
Yes, actually I was about to ask if we need to mention this.